examples/gatsby-theme-docs/gatsby-config.js

```diff
@@ -20,6 +20,11 @@ module.exports = {
         yamlFilesPath: `src/yamlFiles`,
         repositoryUrl: `https://github.com/jpedroschmitz/rocketdocs`,
         baseDir: `examples/gatsby-theme-docs`,
+        gatsbyRemarkPlugins: [
+          {
+            resolve: require.resolve(`./plugins/gatsby-remark-purple-headers`),
+          },
+        ],
```

Comment:
It's nice that you added a local plugin, but it will change all H1 to purple on the website, and we don't want that.

Also, I believe most users don't want a plugin that changes the heading's text color.